Makefile

```diff
@@ -102,11 +102,28 @@ bump:
 ship:
 		pod spec lint --allow-warnings DatadogSDK.podspec
 		pod trunk push --allow-warnings --synchronous DatadogSDK.podspec
-		pod repo update
 		./tools/standalone-binary-distro/make_distro_builds.sh
-		@echo "⚠️ DatadogSDKObjc.podspec needs to be tried after ~1 hour:"
-		@echo "pod spec lint --allow-warnings DatadogSDKObjc.podspec"
-		@echo "pod trunk push --allow-warnings DatadogSDKObjc.podspec"
+ifeq (${ci}, true)
+		curl https://app.bitrise.io/app/$$BITRISE_APP_SLUG/build/start.json \
+		--data '{"hook_info":{"type":"bitrise","build_trigger_token":"$$BITRISE_PERSONAL_ACCESS_TOKEN"},\
+		"build_params":{"workflow_id":"tagged_commit_part_2","commit_hash":"$$BITRISE_GIT_COMMIT"},\
+		"triggered_by":"curl"}'
+endif
+
+ship_part_2:
+		pod trunk me
+		pod spec lint --allow-warnings DatadogSDKObjc.podspec
+		pod spec lint --allow-warnings DatadogSDKAlamofireExtension.podspec
+		objcRetVal=1 ; \
+		while [[ $$objcRetVal -ne 0 ]] ; do \
+			pod trunk push --allow-warnings DatadogSDKObjc.podspec ; \
+        	((objcRetVal = $$?)) ; \
+    	done
+		alamofireRetVal=1 ; \
+		while [[ $$alamofireRetVal -ne 0 ]] ; do \
+			pod trunk push --allow-warnings DatadogSDKAlamofireExtension.podspec ; \
+        	((alamofireRetVal = $$?)) ; \
+    	done
```

Comment:
Does this unusual `done` indentation and line breaks (`\`) come from running the shell script inside `Makefile`? 

If yes, then it will be very hard and error-prone to maintain this script even for a small changes as no dry run can be performed for this automation. Let's at least move it to separate shell script, where the syntax will simpler.

It might be a clear candidate for `distribution` toolset, along with the `make_distro_builds.sh` tool:
```shell
/tools/distribution
    - make_distro_builds.sh
    - push_podspecs.sh
```